apps/storybook/stories/documentation/bidra/Bygge.mdx

```diff
@@ -0,0 +1,98 @@
+import { Alert, AlertIcon, KvibProvider } from "@kvib/react/src";
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Info/Bidra/Bygge" />
+
+## Bygge
+
+### Bygge komponenten 🛠
+
+Under følger en beskrivelse av hvordan en komponent kan bygges.
+I de fleste tilfeller innebærer dette bare å justere på props, men i enkelte tilfeller kan det være ønskelig å bygge selve react-komponenten litt annerledes enn Chakra har gjort.
+Link-komponenten brukes som eksempel under.
+
+1.  I `packages/react/src`: finn mappen med navnet på komponenten, dvs link.
+2.  I denne mappen (`packages/react/src/link`) lag én fil, _Link.tsx_ (i tillegg til _index.tsx_ som ligger der fra før).
+    _Link.tsx_ skal brukes for å kvibifisere komponenten med f.eks. egne props, og index.tsx brukes for å eksportere alt fra _.tsx_-filen.
+3.  I _Link.tsx_: Begynn med å definere hvilke props som skal være gjeldende for din komponent, samt hvilke typer hver prop kan ta. Dette kan gjøres på to måter:
+
+- **Alternativ 1- bygg på Chakras props**: Importer Chakra sine props (her; LinkProps), og fjern unødvendige props og legg evt. til egndefinerte props.
+  Du må også definere en prop fra Chakra på nytt dersom du skal endre på hvilke verdier denne propen kan ta.
+  I koden under har vi fjernet chakra sin egen `variant`, og heller lagt til vår egendefinerte `variant` som kun tar verdiene “green” og “blue”.
+
+            ```jsx
+            import {LinkProps as ChakraLinkProps} from "@chakra-ui/react";
+
+            export type LinkProps = Omit < ChakraLinkProps, "variant" >
+            & {colorScheme: "green" | "blue";};
+            ```
+
+- **Alternativ 2- definer egne props fra bunnen**: Her må man definere alle props fra bunnen av.
+
+        ```jsx
+        export type LinkProps = {
+            colorScheme? : "green" | "blue", //denne er ikke required pga. "?"
+            href: string, //denne er required. Kommer feilmelding dersom denne ikke settes i komponenten.
+            children? : string,
+        };
+        ```
+
+<KvibProvider>
+  <Alert status="info">
+    <AlertIcon />
+    _required_ props er ikke det samme som å sette default. Eksempelvis colorScheme er ikke required fordi man ønsker ikke
+    at brukeren må sette colorScheme dersom hen vil ha default-verdien. Hvordan sette default-verdi kommer i senere steg
+    i denne guiden.
+  </Alert>
+</KvibProvider>
+
+4.  Videre i _Link.tsx_ skal du bygge opp komponenten ved å bruke propsa du definerte i steg 3. De propsene du endrer på sendes manuelt videre til Chakra-komponenten, de andre kan videresendes ved å bruke _“…props”_- syntaksen.
+    Ved å tilegne props verdier settes default for komponenten, noe som må gjøres dersom propsen er satt som required.
```

Comment:
Dette stemmer ikke helt. Dersom en prop er required trenger den ikke å ha default, men det er ofte kjekt at den har det. Dersom en prop er required, men ikke har en default vil brukerne av komponenten måtte sette denne selv for å ikke få feilmelding.